src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -948,15 +948,23 @@ bool SotaUptaneClient::putManifestSimple(const Json::Value &custom) {
     return false;
   }
 
+  static bool connect = true;
   auto manifest = AssembleManifest();
   if (custom != Json::nullValue) {
     manifest["custom"] = custom;
   }
   auto signed_manifest = uptane_manifest.signManifest(manifest);
   HttpResponse response = http->put(config.uptane.director_server + "/manifest", signed_manifest);
   if (response.isOk()) {
+    if (!connect) {
+      LOG_INFO << "Connectivity is restored.";
+      sendEvent<event::LogConnectivityRestored>();
```

Comment:
I think just logging is enough. This is probably easier than we were thinking. Sending an event isn't a bad idea but for now we can skip it.